pkg/steps/source.go

```diff
@@ -314,6 +314,10 @@ func isInfraReason(reason buildapi.StatusReason) bool {
 	return false
 }
 
+func hintsAtInfraReason(logSnippet string) bool {
+	return strings.Contains(logSnippet, "no such image")
```

Comment:
`error: build error: no such image` would be more precise/less likely to catch a false positive